client/Lykke.Service.PayInternal.Client/Models/PaymentRequest/CreatePaymentRequestModel.cs

```diff
@@ -16,6 +16,11 @@ public class CreatePaymentRequestModel
         /// Gets or sets the amount.
         /// </summary>
         public double Amount { get; set; }
+
+        /// <summary>
+        /// Gets or sets the external order id
+        /// </summary>
+        public string OrderId { get; set; }
```

Comment:
The order does not exist until payment request not created. As I know order created when called checkout action.

Reply:
This is another OrderId value. It is like external order id, id from customer.